src/Stomp/Network/Connection.php

```diff
@@ -110,14 +117,16 @@ class Connection
      *
      * @param string $brokerUri
      * @param integer $connectionTimeout in seconds
-     * @param array   $context stream context
+     * @param array $context stream context
+     * @param bool $persistentConnection
      * @throws ConnectionException
      */
-    public function __construct($brokerUri, $connectionTimeout = 1, array $context = [])
+    public function __construct($brokerUri, $connectionTimeout = 1, array $context = [], $persistentConnection = false)
```

Comment:
I see no more flags which would be usefull.
the benefit of having dedicated setters is, that the Connection class api hides the underlying implementation details.